docs/open-api.yaml

```diff
@@ -128,6 +160,27 @@ components:
         sameAs:
           type: string
           nullable: true
+    CreateJournal:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the journal
+        pIssn:
+          type: string
+          nullable: true
+          description: The publication ISSN
+          pattern: ^[0-9]{4}-[0-9]{3}[0-9X]$
+        eIssn:
+          type: string
+          nullable: true
+          description: The electronic ISSN
+          pattern: ^[0-9]{4}-[0-9]{3}[0-9X]$
+        url:
```

Comment:
The URL is not the `sameAs`, it is in my understanding the URL the homepage of the journal, like e.g. "https://adressa.no". Po wants us to allow users to provide the "homepage" on creation, but we don't really need it when fetching the journal. Maybe we should ask them to add it so that we are at least able to return the data that was created. The `sameAs` is just the "Kanalregisterets" representation of the journal.

Reply:
So we take in a homepage in and send to the client, then the client returns a referense url, sameAs, and we return that to our response.